app/sass/objects/_buttons.scss

```diff
@@ -1,4 +1,3 @@
 .btn {
-  border: solid $inuit-btn-border-width $base-color;
-  padding: 15px 40px;
+  padding: 13px 0;
```

Comment:
We should try to avoid `magic numbers` and use framework's variables instead. You have at your disposal `$inuit-base-spacing-unit` and it's variants (small, tiny, large, huge) and those should be used wherever possible.

https://css-tricks.com/magic-numbers-in-css/